services/app/app/components/views/quest/cardtemplates/decision/Scenarios.tsx

```diff
@@ -34,6 +99,16 @@ const scenarios: ScenarioType[] = [
     nonevent: {type: 'INTERRUPTED', text: 'The noise of battle shakes you out of your reverie... back to fighting!', instructions: []},
     retry: {type: 'RETRY', text: 'The thought remains, just out of reach...', instructions: []},
   },
+  simpleScenario(
+    'Light',
+    'Knowledge',
+    'It feels like in a dream, you\'ve been here before...',
```

Comment:
`It feels like you've been here in a dream before...`

Reply:
done